src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -90,25 +102,29 @@ private class Indexing(
           val sparqlView: View             = SparqlView.default(project.ref)
           val resolver: Resolver           = InProjectResolver.default(project.ref)
           // format: off
-          cache.project.replace(project) *>
-            coordinator.start(project) *>
-            resources.create(Id(project.ref, elasticSearchView.id), viewRef, asJson(elasticSearchView)).value.mapRetry(createdOrExists, KgError.InternalError(s"Couldn't create default ElasticSearch view for project '${project.ref}'"): KgError) *>
-            resources.create(Id(project.ref, sparqlView.id), viewRef, asJson(sparqlView)).value.mapRetry(createdOrExists, KgError.InternalError(s"Couldn't create default Sparql view for project '${project.ref}'"): KgError) *>
-            resources.create(Id(project.ref, resolver.id), resolverRef, asJson(resolver)).value.mapRetry(createdOrExists, KgError.InternalError(s"Couldn't create default InProject resolver for project '${project.ref}'"): KgError) *>
-            Task.unit
-          // format: on
+          for {
```

Comment:
while `flatMap` is better than `*>` we should consider doing the first 2 sequentially flatMap the rest in parallel